dev-php/composer/composer-1.2.2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Dependency Manager for PHP"
+HOMEPAGE="https://github.com/composer/composer"
+
+SRC_URI="https://github.com/composer/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-lang/php:*
+	dev-lang/php[curl]
+	dev-php/fedora-autoloader
+	=dev-php/json-schema-2.0.0
```

Comment:
`=` deps are almost always a very bad idea. Please at least use `~` to account for revision bumps.